Portfolio/index.html

```diff
@@ -6,8 +6,66 @@
     <link rel="stylesheet" href="index.css" />
   </head>
 
-  <body style="margin: 0; padding: 0">
-    <!-- Write your code below for PORTFOLIO -->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
+  <body style="margin: 2; padding: 0">
+    <h1>Doe.</h1>
+    <section>
+      <ul>
+        <li>ABOUT</li>
+        <li>SKILLS</li>
+        <li>PROJECTS</li>
+        <li>REVIEW</li>
+      </ul>
+    </section>
+    <img src="/phone.icon.png" alt="phoneicon"><br><br>
+    <span>
+      <h4>JohanDoe</h4>
+      <img src="/johnDoe.png" alt="pic"><br>
+      <strong>Front End Developer</strong>
+    </span>
+<h5>GET KNOW ME</h5>
+<p>i'm a Front-end Developer with a passion for crafting exceptional digital experiences with over 2 years of hands-on experience,I bring a deep proficiency in HTML ,CSS,javaScript,Typescript,and a keen mastery of modern frameworks like React.js,React Native.I specialize in building responsive .interactive and visually appearing user interfaces.Additionally,my proficiency in firebase ensures secure and efficient backend integration ,making me a versatile and dynamic developer ready to tackle any project</p><pre>
+Name      : john Doe
+Age       : 25
+Experience: 2years
+</pre>
+<section>
+  <h2><u>SKILLS</u></h2>
+  <img src="/css.icon.png" alt="css symbol"><br><br>
+  <img src="/js.icon.png" alt="js icon"><br><br>
+  <img src="/html.icon.png" alt="html icon"><br><br>
+  <img src="/svelte.icon.png" alt="sevelte icon"><br><br>
+  <img src="/ts.icon.png" alt="typescript icon"><br><br>
+  <img src="/react.icon.png" alt="react icon">
+  
+</section>
+<section>
+  <form action="get">
+    <h4>Get in Touch</h3>
+      <label for="name">Name :</label>
+      <input type="text" id="name" placeholder="enter name"><br><br>
+      <label for="mail">E-mail :</label>
+      <input type="text" id="mail" placeholder="enter e-mail">
+      <br><br>
+      <label for="message">Message :</label>
+      <textarea name="message" id="message" cols="30" rows="5" placeholder="Add Comments..."></textarea>
+      <input type="button" value="SUBMIT" >
+  </form>
+  <br>
+  <img src="/phone.icon.png" alt="phone icon"> <br><br>
+  <a href="teleto: 7289973774">+917289973774</a>
+  <br><br>
+  <span><a href=" mail to: johndoe" >Mail -:</a> </span>
+  <p>manish23@gmail.com </p>
+  <img src="/linkedin.icon.png"  alt="linkedin icon">
+  <img src="/instagram.icon.png"  alt="instagram">
+  <img src="/twitterX.icon.png"  alt="twitter">
```

Comment:
The opening tag for the second paragraph element should be lowercase 'p' instead of uppercase 'P'.